admin/sql/create_tables.sql

```diff
@@ -67,7 +67,17 @@ CREATE TABLE revision (
     id          SERIAL      NOT NULL,
     review_id   UUID,
     "timestamp" TIMESTAMP   NOT NULL,
-    text        VARCHAR     NOT NULL
+    text        VARCHAR,     
+    rating      SMALLINT CHECK (rating >= 0 AND rating <= 100)
```

Comment:
They exist, but not in huge numbers (436 release group ratings, 708 recording ratings not on a scale of 1-5).

MB's CSS will display fractional stars just fine (it's needed for the average ratings), though if you click on the stars it can only do 1-5. That said, I don't think that makes 0-100 "incorrect"; we could just as easily decide the number of stars we display is incorrect, or trivially modify the UI to support setting half-stars, etc.

Note that it's possible in MB to give something 0 stars, so there are 101 possible ratings.